src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,65 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Long FIRST_DRIVER_ID = 1L;
```

Comment:
Better not use hardcoded ids.
To test your functionality - create new manufacturers, drivers and cars in method 'main'
Do not use hardcoded ids